packages/Tokens/src/tokens.js

```diff
@@ -48,10 +48,10 @@ const tokens = {
     "purple": "#785cba",
     "purpleDarken10": "#5f449f",
     "purpleDarken20": "#4a347c",
-    "purpleLighten10": "#9680c9",
-    "purpleLighten20": "#b3a4d8",
-    "purpleLighten30": "#d1c7e8",
-    "purpleLighten40": "#efebf7",
+    "purpleLighten10": "#9884c5",
```

Comment:
I think got versioned at some point and are installed when doing yarn because Lerna `pretrasnpile` create them. I will remove them from git history so the `.gitignore` can do their job.